pmd.properties

```diff
@@ -1,15 +1,21 @@
 category=External Analysers
 description=Provide PMD rules to analyse Java projects
-archivedVersions=
-#publicVersions=3.3.1
+archivedVersions=3.3.1
+publicVersions=3.4.0
 
 defaults.mavenGroupId=org.sonarsource.pmd
 defaults.mavenArtifactId=sonar-pmd-plugin
 
+3.4.0.description=Upgrade to PMD 6.45 and Java 18 support
+3.4.0.sqVersions=[8.9,LATEST]
+3.4.0.date=2022-05-11
+3.4.0.changelogUrl=https://github.com/jborgers/sonar-pmd/releases/tag/3.4.0
+3.4.0..downloadUrl=https://github.com/jborgers/sonar-pmd/releases/download/3.4.0/sonar-pmd-plugin-3.4.0.jar
+
 3.3.1.description=Upgrade to PMD 6.30 and Java 15 support
-#3.3.1.sqVersions=[8.9,LATEST]
+3.3.1.sqVersions=[8.9,LATEST]
```

Comment:
Only one version can be compatible with `LATEST` at a time. 
When archiving the previous version, you need to update the compatibility range to end with the current version of SonarQube. So 9.4 in this case.